### PR TITLE
dupRadar

### PR DESCRIPTION
**Pull request by siebrenf**
**What's new?**
add dupRadar to the RNA-seq MultiQC

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [ ] I updated the CHANGELOG
- [ ] These changes are covered by the tests

### CODE REVIEW

**Comment by siebrenf** [comment]
TODO:
- extract sample strandedness (they complain if you say paired ended, but the sample is single ended, so why do I have to tell them!?)
- benchmark RAM usage
- stop removing duplicate reads in RNA-seq
- add test
- update docs & changelog

**Comment by siebrenf** [comment]
dups are removed here, but its the thought that counts!

GRCz11-GSM1483769_mqc

**Comment by siebrenf** [comment]
afbeelding
(seems our test data is quite bad!)

- the rule takes <500 MB ram with these samples. Too low to care for resource logging I guess.
- dupRadar will run in the RNA-seq MultiQC tests, so it's covered.
- since the module isn't in MultiQC, each png is added as a separate file.
- since each file has it's sample name, I cant actually add any config options. I've added the "dupRadar" prefix to group them together:
afbeelding

File: seq2science/rules/configuration_explain.smk [hunk 1]
```diff
@@ -73,7 +73,7 @@ else:
                       end=" and finally were tn5 bias shifted by seq2science." if config.get("tn5_shift", 0) > 0 else "."),
         # "samtools_sort": "Bam files were sorted with samtools v@samtools[samtools].",
         "sambamba_sort": "Bam files were sorted with sambamba v@sambamba[sambamba] (https://doi.org/10.1093/bioinformatics/btv098).",
-        "mark_duplicates": "Afterwards, duplicate reads were removed with picard MarkDuplicates v@picard[picard] (http://broadinstitute.github.io/picard).",
+        "mark_duplicates": "Afterwards, duplicate reads were " + ("removed" if "REMOVE_DUPLICATES=true" in config.get("markduplicates", "") else "marked") + " with picard MarkDuplicates v@picard[picard] (http://broadinstitute.github.io/picard).",
```
Comment (Maarten-vd-Sande) [inline seq2science/rules/configuration_explain.smk:76]: I am not sure, but I think most/some tools will not consider reads marked as duplicates. But from the comments with pictures seems that is not an issue actually :thinking:

**Review by Maarten-vd-Sande — APPROVED** [review]
My general idea of the multiqc report is that it starts at the beginning of the analysis (e.g. trimming), and ends with the later steps (perhaps other way around makes more sense though?). Anyways, ideally these plots would be added at the bottom. 

The multiqc config is basically 1 big f string, so you should be able to add them. Take a look at e.g. `macs2_img1`.

File: seq2science/rules/configuration_explain.smk (hunk 1 above)
Comment (siebrenf) [reply seq2science/rules/configuration_explain.smk:76]: dupRadar actually needs reads marked

Comment (Maarten-vd-Sande) [reply seq2science/rules/configuration_explain.smk:76]: yeah makes sense actually :see_no_evil:

**Pull request merged**